config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml

```diff
@@ -130,11 +72,8 @@ spec:
          edpm_ovn_metadata_agent_metadata_agent_DEFAULT_nova_metadata_host: 127.0.0.1
          edpm_ovn_metadata_agent_metadata_agent_DEFAULT_metadata_proxy_shared_secret: 12345678
          edpm_ovn_metadata_agent_DEFAULT_bind_host: 127.0.0.1
-         ctlplane_dns_nameservers:
-         - 192.168.111.1
-         dns_search_domains: []
          edpm_ovn_dbs:
-         - 192.168.111.1
+         - 192.168.122.1
```

Comment:
I think I had issues with `192.168.122.0/24` which is why I ended up on `192.168.111.0/24` I notice the rdo checks haven't run for this change so we might not be seeing the issue with addressing. Provided this actually works in CI and doesn't require some other change, I have no other preference to any particular subnet. Just calling it out in-case there's more required for this subnet change.

Reply:
I think this is another thing that is different when baremetal or not: https://github.com/openstack-k8s-operators/dataplane-operator/commit/9b19caf25ca568012a20738b2b5a12c9579ad4b4